docker/build.sh

```diff
@@ -21,6 +21,8 @@ EOF
 # Enable strict evaluation semantics
 set -e
```

Comment:
would it be useful to set `x` and/or `u` here as well?

Reply:
I explicitly left out the `x`, as it would log the artifactory password; and this script already has decent `echo` statements. We could add it later if we find it's still hard to debug?

Reply:
Hah! Good call then. I think it's fine as-is then